client/js/index.jsx

```diff
@@ -17,11 +17,13 @@ import { setExamples } from "./redux/examples/examplesActions.js";
 import Maybe from "folktale/maybe";
 import Result from "folktale/result";
 import Task from "folktale/concurrency/task";
+import heroes from "heroes";
 
 window.Maybe = Maybe;
 window.Result = Result;
 window.Task = Task;
 window.R = R;
+window.heroes = heroes;
```

Comment:
What is this for?

Reply:
Just so you can explore the data in the console.  The other idea I was thinking of doing was having a json version of it at `/data/heroes.json` so you could open it in a new tab, but I couldn't think of a good way to have a single source of truth without introducing a node server.